report/finished/chapters/modeling.tex

```diff
@@ -1,6 +1,206 @@
+\begin{equation}
+    T = 
+    \begin{bmatrix}
+        R & d \\
+        \bf{0} & 1
+    \end{bmatrix}
+\end{equation}
+where \(R\) is a 3 x 3 rotation matrix and \(d\) a 3 x 1 translation matrix. This implies that any transformation could be characterized by six parameters, three for the translation and three for the rotation. \cite{spong}
+
+
+\subsection*{Denavit-Hartenberg Convention}
+A common approach for selecting the coordinate frames of reference for each joint in a robotic arm is to us the Denavit-Hartenberg convention. This allows the transformation matrix for each joint to be expressed as
+\begin{equation}
+    A_i = Rot_{z}(\theta_i) \cdot Trans_{z}(d_i) \cdot Trans_{x}(a_i) \cdot Rot_{z}(\alpha_i)
+    \label{eqn:DH-transformation}
+\end{equation}
+that consists of the four basic transformations
+\begin{equation}
+    Rot_z(\theta_i) = 
+    \begin{bmatrix}
+        cos(\theta_i) & -sin(\theta_i) & 0 & 0 \\
+        sin(\theta_i) & cos(\theta_i) & 0 & 0 \\
+        0 & 0 & 1 & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Trans_z(d_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & 1 & 0 & 0 \\
+        0 & 0 & 1 & d_i \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Trans_x(\alpha_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & \alpha_i \\
+        0 & 1 & 0 & 0 \\
+        0 & 0 & 1 & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}
+\end{equation}
+\begin{equation}
+    Rot_z(\theta_i) = 
+    \begin{bmatrix}
+        1 & 0 & 0 & 0 \\
+        0 & cos(\alpha_i) & -sin(\alpha_i) & 0 \\
+        0 & sin(\alpha_i) & cos(\alpha_i) & 0 \\
+        0 & 0 & 0 & 1
+    \end{bmatrix}.
+\end{equation}
+
+
+Where the parameters \(\theta_i\), \(a_i\), \(d_i\) and \(\alpha_i\), known as DH-parameters, characterize each joint. To allow the transformation for each joint to be represented by only four parameters, compared to the six parameters that is required in general, there are some restrictions on how the coordinate frames can be chosen. To comply with the DH-convention, the following must be satisfied.
+\begin{itemize}
+    \item The axis \(x_{1}\) is perpendicular to the axis \(z_0\)
+    \item The axis \(x_{1}\) intersects the axis \(z_0\)
+\end{itemize}
+where it is assumed that two frames are given, frame 0 and frame 1, and the transformation from equation \ref{eqn:DH-transformation} transforms a coordinate from frame 1 into a coordinate in frame 0.\cite{spong}
+
+
+
+\subsection*{Kinematic Chain} %A1*A2*...
+A robotic manipulator can be described by a set of joints with links between them where a homogeneous transformation matrix \(A_i\) that describes the transformation with respect to the previous joint exists for each joint. This means that a transformation that describes the position and orientation of joint j with respect to a joint i to a can be found by a transformation matrix \cite{spong}
```

Comment:
Don't need the "that" in "\(A_i\) that describes the ..."
Might be a good idea to use \textit{} or $ $  when talking about the joints and matrices. It will make it clearer for the reader that these letters are notations for the joints.

Reply:
Maybe the sentence might be a bit too long, but removing the "that" would make it wrong. Maybe it would look better if the "that describes the transformation with respect to the previous joint" would be inside comma signs?
@mnsnor 

$ $ means the same as \( \). And yes, I will add them to "i" and "j" as well.

Reply:
No, it's wrong with the "that", otherwise this section "them where a homogeneous..." have the where in it